### PR TITLE
Checks for Obsoletes

### PR DESCRIPTION
**Pull request by ignatenkobrain**

### CODE REVIEW

File: build/parseReqs.c [hunk 1]
```diff
@@ -277,6 +277,21 @@ rpmRC parseRCPOT(rpmSpec spec, Package pkg, const char *field, rpmTagVal tagN,
...
+		goto exit;
+	    }
+	    if (!EVR) {
+		rasprintf(&emsg, _("It's not recommended to have "
+				   "unversioned Obsoletes"));
+	    } else if (Flags & RPMSENSE_GREATER) {
+		rasprintf(&emsg, _("It's not recommended to use "
+				   "'>' in Obsoletes"));
+	    }
+	}
```
Comment (Conan-Kudo) [inline build/parseReqs.c]: These are just warnings here, right?

Comment (ignatenkobrain) [reply build/parseReqs.c]: yes, unless you use goto

**Comment by jasontibbitts** [comment]
These are pretty minor but they do all seem to me to be reasonably good ideas.

**Comment by n3npq** [comment]
Nagware to effect policy is usually just an annoyance.

There are arguable cases (in renaming) where versionless or greater than are perfectly sensible.

Also: Obsoletes: (after installation) are essentially Conflicts: and should be treated similarly if you insist on adding nagware.

**Pull request merged**